core/sql/optimizer/NormRelExpr.cpp

```diff
@@ -7050,8 +7077,8 @@ void RelRoot::recomputeOuterReferences()
       child(0)->getGroupAttr()->getCharacteristicInputs().
                   getLeafValuesForCoverTest(leafValues, emptyGA, emptySet);
       CMPASSERT((getGroupAttr()->getCharacteristicInputs().contains
-                (child(0)->getGroupAttr()->getCharacteristicInputs())) || 
-                (getGroupAttr()->getCharacteristicInputs().contains (leafValues)));
+		 (child(0)->getGroupAttr()->getCharacteristicInputs())) || 
```

Comment:
Message from the TAB police, if you can, configure your editor to put spaces instead of tabs. The good news is that they are configured as 8 spaces :-)